src/rf/js/src/core/models.js

```diff
@@ -60,7 +60,7 @@ var Layer = Backbone.Model.extend({
         return _.findWhere(this.get('images'), {id: id});
     },
 
-    isUploading: function() {
+    isPreValidated: function() {
```

Comment:
Prevalidated seems like the wrong term here. I found it confusing considering we have an image validation step. Maybe `isStarted()` would be better.